pysb/kappa.py

```diff
@@ -138,8 +141,12 @@ def run_simulation(model, time=10000, points=200, cleanup=True,
     fm_filename = base_filename + '_fm.dot'
     out_filename = base_filename + '.out'
 
-    args = ['-i', kappa_filename, '-t', str(time), '-p', str(points),
-            '-o', out_filename]
+    if points == 0:
+        raise ValueError('The number of data points cannot be zero.')
+    plot_period = (1.0 * time / points) if time > 0 else 1.0
```

Comment:
I assume the `1.0 *` is to force float division? Prefer `float(time) / points` **or** `from __future__ import division` at the top of the module (I just checked and there's no integer division in this module, in fact there's no other division at all).